scripts-available/CDB_DistType.sql

```diff
@@ -0,0 +1,138 @@
+--
+-- CDB_DistType classifies the histograms of a column into
+-- one of the basic types listed by Galtong: http://druedin.com/2012/12/08/galtungs-ajus-system/
+-- 
+-- Future improvements:
+--    variable number of bins (7 is baked in right now)
+--    catch the number of items to ensure that the sample is large enough
+--
+-- Refs:
+--    1. width_bucket/histograms: http://tapoueh.org/blog/2014/02/21-PostgreSQL-histogram
+--    2. R implementation: https://github.com/cran/agrmt
+
+CREATE OR REPLACE FUNCTION CDB_DistType ( in_array NUMERIC[] ) RETURNS text as $$
```

Comment:
@javisantana mentioned that I should use the histogram in pg_stats instead. That seems like it would solve this problem?